src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/watasks/controllers/GetTaskByIdControllerTest.java

```diff
@@ -140,8 +211,10 @@ public void should_return_a_200_with_task_when_next_hearing_date_is_null() {
             .body("task.warnings", equalTo(false))
             .body("task.case_management_category", equalTo("Protection"))
             .body("task.work_type_id", equalTo("hearing_work"))
-            .body("task.permissions.values", equalToObject(List.of("Read", "Refer", "Execute")))
-            .body("task.description", equalTo("[Decide an application](/case/WA/WaCaseType/${[CASE_REFERENCE]}/"
+            .body("task.permissions.values",
+                  equalToObject(List.of("Read", "Own")))
```

Comment:
define `List.of("Read", "Own")` as a variable an reuse, instead of re declaring ?

Reply:
Variable declared